bundles/io/org.eclipse.smarthome.io.transport.mqtt.test/src/test/java/org/eclipse/smarthome/io/transport/mqtt/test/MqttServiceTests.java

```diff
@@ -19,75 +21,60 @@
 import org.eclipse.smarthome.io.transport.mqtt.MqttBrokerConnection;
 import org.eclipse.smarthome.io.transport.mqtt.MqttBrokersObserver;
 import org.eclipse.smarthome.io.transport.mqtt.MqttService;
+import org.junit.Before;
 import org.junit.Test;
+import org.mockito.MockitoAnnotations;
 
 /**
  * Tests the MqttService class
  *
  * @author David Graeff - Initial contribution
  */
 public class MqttServiceTests {
-    boolean brokerExisting = false;
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
You can remove `MockitoAnnotations.initMocks(this)` here because you dont have annotated mocks. Your mock in line 41 is created instant. The line in setup is necessary if you have instance members which are mocks like

```
@Mock
private MqttBrokersObserver observer;
```

This way `observer` will always be present. You might go this way if you use your mock in more than one test. So ne need to change it here.